frontend/src/pages/documents/documents.vuex-module.ts

```diff
@@ -12,11 +12,53 @@ class DocumentsVuexModule extends VuexModule {
     this.documents = documents;
   }
 
+  @Mutation
+  addDocument(document: DocumentDto): void {
+    this.documents.push(document);
+  }
+
+  @Mutation
+  removeDocument(index: number): void {
+    this.documents.splice(index, 1);
+  }
+
+  @Mutation
+  appendContributor(tuple: [number, string]): void {
+    const document = this.documents[tuple[0]];
+    document.contributors.push(tuple[1]);
+  }
+
+  @Mutation
+  editModifiedDate(tuple: [number, string]): void {
+    const document = this.documents[tuple[0]];
+    document.modifiedDate = tuple[1];
+  }
+
```

Comment:
Actions related to edited document should be moved to edit-document.vuex-module.ts (see develop)

Reply:
Sorry, not develop, feature/ws-rooms